gestionatr/input/messages/message.py

```diff
@@ -146,7 +149,7 @@ def __init__(self, xml, force_tipus=None):
     @staticmethod
     def check_fpos(f_obj):
         """Setejar la posició actual dels fixers"""
-        if isinstance(f_obj, file) and f_obj.tell() != 0:
+        if isinstance(f_obj, IOBase) and f_obj.tell() != 0:
```

Comment:
This validation not require needs the same validation `(six.PY2 and isinstance(f_obj, file))` ?